### PR TITLE
remove forum, github, share links in china

### PR DESCRIPTION
**Pull request by wjllance**
- GitHub.   some of our China customer complain about our open source code, so we'd better remove github links.
- Forum， currently it is from US server. I afraid it will be covered every time from US update. May be blocked  also. So did social medias.

### CODE REVIEW

File: app/templates/base.jade [hunk 1]
```diff
@@ -86,7 +87,7 @@ block footer
       if me.isAdmin()
         a(href='/admin') Admin
 
-      if usesSocialMedia && me.useSocialSignOn()
+      if usesSocialMedia && me.useSocialSignOn() && !features.china
```
Comment (AndrewJakubowicz) [inline app/templates/base.jade]: Should `features.china` be added to the `useSocialSignOn` method?
I am not sure you need this addition as `useSocialSignOn` is the inverse of `features.chinaUx`, which is correctly set on koudashijie?

File: app/templates/base.jade [hunk 2]
```diff
@@ -54,7 +55,7 @@ block header
                 if !features.codePlay
                   li
                     a#logout-button(data-i18n="login.log_out")
-  
+
```
Comment (AndrewJakubowicz) [inline app/templates/base.jade]: Great white space cleanup! :)

File: app/templates/base-flat.jade [hunk 3]
```diff
@@ -71,8 +71,9 @@ mixin accountLinks
                     if !me.isAnonymous() && !me.isStudent() && !me.isTeacher()
                       li
                         a.text-p(href="/play", data-i18n="common.play")
-                      li
-                        a.text-p(href=view.forumLink(), data-i18n="nav.forum")
+                      if !features.china
```
Comment (AndrewJakubowicz) [inline app/templates/base-flat.jade]: Is there a reason why we're not using a method on the `me` object?

**Comment by wjllance** [comment]
> Instead of using `features.china`, why aren't we using an existing method such as `me.useSocialSignOn`?
> It is probably not the right method for all these changes. But we shouldn't directly use `features.china`, and instead use a descriptive method on `me`.

I see. I will pay more attention next time

**Comment by AndrewJakubowicz** [comment]
Thanks for making the changes.
The build is breaking because static pages use mocked methods to work.
The `me.showChinaICPinfo()` method doesn't exist on the [static-mock](https://github.com/codecombat/codecombat/blob/2ea48c9c67f0c733c4766befdd7ee65be05cb216/static-mock.coffee#L13).

**Pull request merged**